Dockerfile.cuda

```diff
@@ -0,0 +1,22 @@
+FROM nvidia/cuda:12.2.0-devel-ubuntu22.04
```

Comment:
check out https://github.com/jmorganca/ollama/pull/500 which adds multi arch to this

Reply:
That PR will go into this one 👍